### DISCUSSION TITLE
When is a release planned to be released?

### DISCUSSION THREAD

**Discussion by RaptaG** [KRTirtho/spotube#412] [General]
Since the last release there have been quite some changes and it would be awesome to get a new release soon, which the additions made so far

**Comment by KRTirtho** [comment]
Yeah, it's been a while since Spotube had a release. Some of the greatest features have been added to Spotube lately
 
Spotube has always been buggy. The "Spotube is quite buggy" is a kind of universal truth now :sweat_smile:. But in this new upcoming release, there've been some major changes in the **playback system**. It was arguably the buggiest part of Spotube. So this release is kind of mission for me to fix those
A major re-write of Playback manager of Spotube has fixed a lot of related bugs, and I can now confidently say at least Spotube's playback isn't buggy anymore. But it took a bit of time to make those changes

You can expect to get a new release by the end of the month

**Reply by RaptaG** [reply]
Nice, I hope that in the future even more "bugged" parts of Spotube will be fixed.

**Reply by KRTirtho** [reply]
As a single developer and maintainer, it's hard to maintain and at the same time develop 4 active projects (`spotube`, [fl-query](https://github.com/KRTirtho/fl-query), [platform_ui](https://github.com/KRTirtho/platform_ui) and [wives](https://github.com/KRTirtho/wives) in case you're wondering)

So, outside contribution from various developers will really help the project to go forward. Although you, @karniv00l and many others have helped a lot. But having more contributors who contribute (code contribution) to the project regularly would help me and the project overall

**Reply by RaptaG** [reply]
The platform UI is (for Spotube) not a necessity but more of a opinionated feature. Improving the original UI is the way to go imo
